diff-pdf: add page (#7711)
tldr-pages/tldr: pages/common/diff-pdf.md

# diff-pdf

> > Tool for visually comparing two PDFs.

Reviewer: ```suggestion
> Tool for comparing two PDFs.
```

Considering that the base command doesn't do this, I think its best to keep it removed.

Author: Also quoted from the creator: "diff-pdf is a tool for visually comparing two PDFs.", but I agree your suggestion is better.

> More information: <https://github.com/vslavik/diff-pdf>.

> - Compare PDFs, (return code 0 = no differences, return code 1 = PDFs differ):

Reviewer: Is there no other indication?

```suggestion
- Compare PDFs using return codes (`0` = no difference, `1` = difference with PDFs)  :
```

Author: Quoted from the creator: "It takes two PDF files as arguments. By default, its only output is its return code, which is 0 if there are no differences and 1 if the two PDFs differ."

Reviewer: Ok. I'll take their word :)

Whats your opinions on the suggestion?

Author: Well, i agree that the brackets or comma should go, but it doesn't "use" return codes to compare the PDFs

Reviewer: Good point, my suggestion has been updated.

> `diff-pdf a.pdf b.pdf`

Reviewer: ```suggestion
`diff-pdf {{path/to/a.pdf}} {{path/to/b.pdf}}`
```

Reviewer: Could you please apply this change throughout.

> - Compare PDFs, output PDF with visually highlighted differences:

Reviewer: ```suggestion
- Compare PDFs outputting a PDF with visually highlighted differences:
```

`diff-pdf --output-diff=diff.pdf a.pdf b.pdf`

> - Compare PDFs, view differences in a simple GUI:

Reviewer: ```suggestion
- Compare PDFs, viewing differences in a simple GUI:
```
